packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.tsx

```diff
@@ -138,77 +99,31 @@ function _QueryWrapper(props: QueryWrapperProps) {
     addValueToSelectedFacet,
     setRangeFacetValue,
     resetDebounceTimer,
+    currentPage,
+    goToPage,
+    pageSize,
+    setPageSize,
```

Comment:
Expose methods that change pagination parameters on the QueryBundleRequest. They can be called anywhere, but will do nothing when rendering infinite row data.